benchmarks/cifar10/cifar.py

```diff
@@ -61,9 +61,9 @@ def __init__(self, root, train=True,
         if download:
             self.download()
 
-        if not self._check_integrity():
-            raise RuntimeError('Dataset not found or corrupted.' +
-                               ' You can use download=True to download it')
+        # if not self._check_integrity():
+        #     raise RuntimeError('Dataset not found or corrupted.' +
+        #                        ' You can use download=True to download it')
```

Comment:
Removed this because it was adding some overhead to PyTorch. They are just verifying if the files are downloaded correctly, which causes a considerable overhead to them.